week1/Homework/array_task5/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿// See https://aka.ms/new-console-template for more information
+int[] input = new[] { 1, 2, 5, 7, 2, 3, 5, 7 };
+int[][] splitArray = Split(input);
+AddArray(splitArray);
+int[] result = AddArray(splitArray);
+WriteResult(result);
+
+int[][] Split(int[] inputArray)
+{
+    //     if (inputArray.Length%2!=0){
+    //         int[][] result= new int [][]{};
+    //         Console.WriteLine(result.Length);
+    //             return new int[][] { };
+    //     }
+    int[] firstArray = new int[inputArray.Length / 2];

```

Comment:
try not to repeat yourself. Create a variable and assign `inputArray.Length / 2` to it and reuse varaible afterwards.